tidb-lightning/tidb-lightning-physical-import-mode-usage.md

```diff
@@ -83,15 +95,46 @@ For the complete configuration file, refer to [the configuration file and comman
+
+Detection before importing has the following limitations:
+
+- Before importing, it reads all data and encodes it to detect potentially conflicting data. Temporary files are stored using `tikv-importer.sorted-kv-dir` during the detection process. The results are retained for import phase after the detection is complete. Therefore, there is additional overhead for time consumption, disk space usage, and API requests to read the data.
+- It only applies to a single node and cannot be applied to parallel import scenarios.
+
+Compared with detection after importing, if the original data has a large amount of conflicting data, using detection before importing can take less import time. It is recommended that you use detection before importing in non-parallel import tasks when the data contains conflicting data and there is sufficient local disk space.
+
+### Detection after importing (the old behaviour)
+
+Detection after importing will be enabled when configuration `tikv-importer.duplicate-resolution` is set. This is the only method of conflict detection in the version prior to v7.2.0.
 
-TiDB Lightning offers three strategies for detecting conflicting data:
+TiDB Lightning offers two strategies for detecting conflicting data:
 
-- `record`: only records conflicting records to the `lightning_task_info.conflict_error_v1` table on the target TiDB. Note that the required version of the target TiKV is v5.2.0 or later versions; otherwise, it falls back to 'none'.
 - `remove` (recommended): records all conflicting records, like the `record` strategy. But it removes all conflicting records from the target table to ensure a consistent state in the target TiDB.
-- `none`: does not detect duplicate records. `none` has the best performance in the three strategies, but might lead to inconsistent data in the target TiDB.
+- `none`: does not detect duplicate records. `none` has the best performance in the two strategies, but might lead to inconsistent data in the target TiDB.
 
-Before v5.3, Lightning does not support conflict detection. If there is conflicting data, the import process fails at the checksum step. When conflict detection is enabled, regardless of the `record` or `remove` strategy, if there is conflicting data, Lightning skips the checksum step (because it always fails).
+Before v5.3, Lightning does not support conflict detection. If there is conflicting data, the import process fails at the checksum step. When conflict detection is enabled, if there is conflicting data, Lightning skips the checksum step (because it always fails).
```

Comment:
```suggestion
Before v5.3, TiDB Lightning does not support conflict detection. If there is conflicting data, the import process fails at the checksum step. When conflict detection is enabled, if there is conflicting data, TiDB Lightning skips the checksum step (because it always fails).
```